include/deal.II/lac/trilinos_tpetra_precondition.h

```diff
@@ -1316,9 +1316,127 @@ namespace LinearAlgebra
       initialize(const SparseMatrix<Number, MemorySpace> &A,
                  const AdditionalData &additional_data = AdditionalData());
     };
-#  endif // DEAL_II_TRILINOS_WITH_IFPACK2
+#    ifdef DEAL_II_TRILINOS_WITH_TPETRA_MUELU
 
-  } // namespace TpetraWrappers
+    /**
+     * @brief The class for the Block SSOR preconditioner within Ifpack2.
```

Comment:
You forgot to update the description here.
Otherwise, it looks already really good!